crates/core/src/view_admin.rs

```diff
@@ -250,6 +250,26 @@ INSERT INTO ps_migration(id, down_migrations) VALUES(3, json_array(json_object('
     ").into_db_result(local_db)?;
     }
 
+    if current_version < 4 {
+        // language=SQLite
+        local_db.exec_safe("\
+ALTER TABLE ps_buckets ADD COLUMN op_checksum INTEGER NOT NULL DEFAULT 0;
+ALTER TABLE ps_buckets ADD COLUMN remove_operations INTEGER NOT NULL DEFAULT 0;
+
+UPDATE ps_buckets SET op_checksum = (
+  SELECT IFNULL(SUM(ps_oplog.hash), 0) & 0xffffffff FROM ps_oplog WHERE ps_oplog.bucket = ps_buckets.name
+);
+
+INSERT INTO ps_migration(id, down_migrations)
```

Comment:
I see this table is only introduced in https://github.com/powersync-ja/powersync-sqlite-core/pull/27/files. Which is probably okay if this is released at the same time.

Reply:
My bad. I was just about to close this after seeing that :)